.github/workflows/cabal.yml

```diff
@@ -9,13 +9,13 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest]
-        ghc: ["9.4", "9.6", "9.8"]
+        ghc: ["9.6", "9.8", "9.10"]
```

Comment:
FYI removing 9.4 from CI, because its associated base 4.17 is out of bounds for ghc-lib-parser, which requires base >= 4.18.